pkg/yurthub/types/constants.go

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Do not create a new sub directory `types`, please remove `const` definition into `sharecache` directory?

Reply:
done